src/language-js/print/function.js

```diff
@@ -254,6 +255,10 @@ function printArrowChain(
 
   const groupId = Symbol("arrow-chain");
 
+  if ((isCallLikeExpression(parent) && !isCallee) || isBinaryish(parent)) {
```

Comment:
Can it be done in `binaryish.js`? Looks little messy to me.

Reply:
I don't think it can be done there. The corresponding indent is located right in this file:

https://github.com/prettier/prettier/blob/9a8ebe6b4a2f3192f96c1eb9ddad8adb95354d61/src/language-js/print/function.js#L269-L275

If we remove the indent added in `binaryish.js`, we'll have output like this:
```js
const weaponizedStellatedOctahedron =
  askTrovenaBeenaDependsRowans ??
  ((
    averredBathersBoxroomBuggyNurl,
    anodyneCondosMalateOverateRetinol = "default",
  ) =>
  (annularCooeedSplicesWalksWayWay) =>
  (kochabCooieGameOnOboleUnweave) =>
      abugidicRomanocastorProvider);
```
instead of expected
```js
const weaponizedStellatedOctahedron =
  askTrovenaBeenaDependsRowans ??
  ((
    averredBathersBoxroomBuggyNurl,
    anodyneCondosMalateOverateRetinol = "default",
  ) =>
    (annularCooeedSplicesWalksWayWay) =>
    (kochabCooieGameOnOboleUnweave) =>
      abugidicRomanocastorProvider);
```

Reply:
I didn't mean to remove `indent` for all, but move `(isCallLikeExpression(parent) && !isCallee)` there, if it matches, don't do `indent`. I'm not familiar with this part, just a question.

Reply:
I understood your question and showed what would happen in that case.

Reply:
Fine.